### DISCUSSION TITLE
Libra in Fedora 35

### DISCUSSION THREAD

**Discussion by vitalyrassolov** [Quantum-Dynamics-Hub/libra-code#157] [Q&A] [answered]
Perhaps it is convenient to put compilation errors/notes into one place.  So far, conda installs 11.3 version of c++, and it gives compilation errors.  Trying to downgrade c++, hope it will fix the problem

**Comment by vitalyrassolov** [comment, marked as answer]
downgrading with **conda install gcc_linux-64=9.3.0** seem to work

**Reply by alexvakimov** [reply]
Thank you, Vitaly!  Yes, this is a great idea to put any problems and potential solutions in one place. 
On the subject - yes, Libra's C++ part uses more conventional (read "old") C++ language standards. No C++11 or  higher yet. Maybe one day.
